drivers/gpio/gpio_pca95xx.c

```diff
@@ -488,6 +650,9 @@ static const struct gpio_pca95xx_config gpio_pca95xx_##inst##_cfg = {	\
 	.common = {							\
 		.port_pin_mask = GPIO_PORT_PIN_MASK_FROM_DT_INST(inst),	\
 	},								\
+	.gpio_port = DT_INST_GPIO_LABEL(inst, int_gpios),               \
```

Comment:
This produces compiler errors when interrupt support is not enabled because the assigned-to fields don't exist.

You'll need to do something like:
```
#ifdef CONFIG_PCA95XX_INTERRUPT_SUPPORT
#define EXPAND_INTERRUPT(inst) \
	.gpio_port = DT_INST_GPIO_LABEL(inst, int_gpios),               \
	.int_pin = DT_INST_GPIO_PIN(inst, int_gpios),                   \
	.int_flags = DT_INST_GPIO_FLAGS(inst, int_gpios),  
#else 
#define EXPAND_INTERRUPT(inst)
#endif 
```